tests/test-cases/Group6-VIC-Machine/6-2-Default.md

```diff
@@ -0,0 +1,18 @@
+Test 2-4 - Verify Help
+=======
+
+#Purpose:
+Verify vic-machine delete default parameters of compute-resource and name
+
+#References:
+* vic-machine-linux delete --target XXX
```

Comment:
Sorry for forgot to update the title. But this is to test default parameters. Will update the title.